411-dmz-example-1/README.md

```diff
@@ -0,0 +1,10 @@
+# Build a DMZ with Network Security Groups (NSG)
```

Comment:
A word on the folder naming for these, I would call these something along these lines that talks about the features in the folder name

* 300-dmz-nsg
* 300-dmz-nsg-firewall
* 300-dmz-nsg-firewall-udr